### DISCUSSION TITLE
source for docker hub images

### DISCUSSION THREAD

**Discussion by clacke** [prisma/prisma#2500] [Q&A] [answered]
Hi, how can I reproduce the docker images up on dockerhub `prismasql/prisma`?

Are the Dockerfile and build scripts available on github? I've looked around with no success.

**Comment by ryands17** [comment, marked as answer]
Hey @clacke :wave: 
Prisma 2 doesn't have any Docker images. As for Prisma 1, you can ask your queries [here](https://github.com/prisma/prisma1) 🚀

**Comment by clacke** [comment]
For others who may see this, the Dockerfile seems to be dynamically generated in https://github.com/prisma/prisma1/blob/56ee4251864b782330865e3e8ebdf89a25e9a0da/server/build.sbt

**Comment by ludralph** [comment]
Hi @clacke,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
